docs/terms-policies/governance.md

```diff
@@ -0,0 +1,242 @@
+# DANDI Archive Project Governance
+
+Version: 1.0  
+Effective Date: YYYY-MM-DD  
+Status: Draft
+
+## 1. Purpose
+
+This document defines the governance structure, roles, responsibilities, and decision‑making processes for the DANDI Archive ecosystem. It applies uniformly to all current and future repositories and services, including:
+
+Sites:
+
+- https://dandiarchive.org
+- https://hub.dandiarchive.org
+- https://about.dandiarchive.org
+
+Code git repositories:
+
+- https://github.com/dandi
+
+Data git/git-annex repositories:
+
+- https://github.com/dandisets
+- https://github.com/dandizarrs
+
+## 2. Mission
+
+DANDI (Distributed Archives for Neurophysiology Data Integration) enables FAIR (Findable, Accessible, Interoperable, Reusable) publishing, preservation, discovery, and computational reuse of neurophysiology data.  DANDI provides:
+
+- A cloud-based platform to store, process, and disseminate data. You can use DANDI to collaborate and publish datasets.
+- Open access to data to enable secondary uses of data outside the intent of the study.
+- Optimize data storage and access through partnerships, compression and accessibility technologies.
+- Enables reproducible practices and publications through data standards such as NWB and BIDS.
+- The platform is not just an endpoint to dump data, it is intended as a living repository that enables collaboration within and across labs.
+
+## 3. Core Principles
+
+1. Openness & Transparency: Designs, discussions, and decisions are public by default
+2. FAIR & Reproducibility: Data and code evolution remain traceable and citable
```

Comment:
```suggestion
2. FAIR & Reproducibility: Data and code follow standards and their evolution remain open, traceable, and citable
```

since IMHO standards are the core of FAIR